src/components/common/admin-dashboard-sidebar/sidebar.css

```diff
@@ -0,0 +1,62 @@
+/* The side navigation menu */
```

Comment:
please edit the sidebar as following : 
1- change the hover color and the background of the sidebar
2- make it display by default , but the admin can hide it when click on the sidebar button